steps/cli/oc.go

```diff
@@ -165,6 +166,16 @@ var _ = gauge.Step("Update addon config with resolverStepActions as <resolverSte
 	}
 })
 
+var _ = gauge.Step("Verify test versioned ecosystem task", func() {
+	taskList := cmd.MustSucceed("oc", "get", "task", "-n", "openshift-pipelines").Stdout()
+	requriedTasks := []string{"git-cli", "git-clone", "s2i-go", "s2i-python"}
+	for _, task := range requriedTasks {
+		if !strings.Contains(taskList, task) {
```

Comment:
You are supposed to check versioned tasks but you don't check those.

Reply:
Done